apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v1/eobByPatientIdAll.json

```diff
@@ -239,7 +239,7 @@
         }
       } ],
       "careTeam" : [ {
-        "sequence" : 2,
+        "sequence" : 1,
```

Comment:
I still don't understand these changes or the ones for care team link. Do the JSONs need to be regenerated?